src/ray/protobuf/direct_actor.proto

```diff
@@ -0,0 +1,32 @@
+syntax = "proto3";
+
+package ray.rpc;
+
+import "src/ray/protobuf/common.proto";
+
+message ReturnObject {
+  // Object ID.
+  bytes object_id = 1;
+  // Data of the object.
+  bytes data = 2;
+  // Metaata of the object.
+  bytes metadata = 3;
+}
+
+message PushTaskRequest {
+  // The ID of the task to be pushed.
+  bytes task_id = 1;
```

Comment:
remove this? because `task_spec` already has `task_id`.

Reply:
actually I'm thinking about restoring `task_spec` to raw bytes instead of a `TaskSpec`, as copying bytes should be much cheaper than copying `TaskSpec`. In that case this field would be needed.

Reply:
where are we copying the task spec now? Can we avoid that by `shared_ptr`? I'd avoid defining task spec as raw bytes if possible.

Reply:
there are a couple of places right now. I ended up modifying `TaskSpecification` to hold a shared_ptr of `TaskSpec`, and do a `Swap` instead of `CopyFrom` for `PushTaskRequest`.

Reply:
we can still use `TaskSpec` in this case, no need to use raw bytes.